### DISCUSSION TITLE
'await' outside function, but it's inside[THIS ISSUE IN ITSELF WAS FIXED]

### DISCUSSION THREAD

**Discussion by Energia360** [nextcord/nextcord#694] [Q&A]
I made this really simple code, but it's not working. Apparently, my await isn't inside an async function. 
```py
import nextcord

from nextcord.ext import commands

client = commands.Bot(command_prefix = '.', intents = discord.Intents().all())

@client.event
async def on_ready():
    print('Kingdom Bot Online')

@client.event
async def on_member_join(member):
    print(f'{member} has joined the server.')

@client.event
async def on_member_remove(member):
    print(f'{member} has left the server.')

@bot.command()
async def ping(ctx):
    await ctx.reply('Pong!')


client.run('my token, not telling ya')
```

The faulty part is after @bot.command. 
```py

    File "/Users/arigayasuo/kingdoms-bot/bot.py", line 18
    await client.process_commands(message)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: 'await' outside function
```

pls help

EDIT: Seems like I just didn't save my new fixed code, sorry for the trouble. Now I have to fix the new error. Says it couldn't access the required privileged intents, I'll make a new thread if I can't seem to fix it. Thanks for the help!

**Comment by EmmmaTech** [comment]
Your indentation is wrong. The `await ctx.reply` line is not in the command definition's scope.

**Reply by Energia360** [reply]
in my atom script it is indented, lemme just fix it

**Comment by Skelmis** [comment]
Why are you using `bot`, when your test code here doesn't use bot at all?

**Reply by Energia360** [reply]
I was following a tutorial, i just started

**Reply by Energia360** [reply]
thanks for the help, it was kinda fixed

**Comment by DenverCoder1** [comment]
First of all, you need to stick to one variable name for your bot. Both `client` and `bot` refer to your Bot instance, but you only assigned the instance to `client`. I would recommend changing this all to `bot` for consistency.
eg.
```py
bot = commands.Bot(command_prefix='.', intents=nextcord.Intents().all())
```
And use `@bot.event` in place of `@client.event`, and `bot.run` in place of `client.run`.

Additionally, you did not include a full traceback of the error so it's hard for anyone else to know where the error occurred.

If you could add the full error message including line numbers, that would help us help you.

**Reply by Energia360** [reply]
Understood. I’m adding the error message.

**Reply by Energia360** [reply]
fixed the syntaxerror, new error debugging

**Comment by ooliver1** [comment]
Your code does not match your provided traceback, there is no `process_commands` at line 18 of the provided code.

**TIP:** use `` ```py `` over `` ``` `` for python codeblocks to add syntax highlighting. I have edited that for you.

**Reply by Energia360** [reply]
I ran the script again, but it still says "line 18". I feel like there's something other than the code messing with me at this point.

**Reply by Energia360** [reply]
was fixed, just didnt save
new error tho

**Comment by DenverCoder1** [comment]
I'd suggest reading this section about privileged Intents:

https://docs.nextcord.dev/en/latest/intents.html#privileged-intents

They must be enabled, not only in the code, but also on the Discord developer portal.
